tests/migration/postcopy.go

```diff
@@ -186,7 +188,7 @@ var _ = SIGMigrationDescribe("VM Post Copy Live Migration", func() {
 					libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
 					libmigration.ConfirmMigrationMode(virtClient, vmi, v1.MigrationPostCopy)
 				},
-					Entry("a migration policy", applyWithMigrationPolicy),
+					Entry("a migration policy", decorators.Conformance, applyWithMigrationPolicy),
```

Comment:
I am hesitant here, 1. The policies are alpha? and can be disabled and 2. the post-copy is off by default, meaning, is this really hard requirement?

Reply:
Maybe not! Dropping it